README.md

```diff
@@ -21,7 +21,7 @@ By default, `npm run build` will generate a Node app that you can run with `npm
 
 ## Tasks
 
-- [ ] setup Prettier (maybe eslint)
```

Comment:
Not sure if we should remove prettier, although it was much more simpler to test without it